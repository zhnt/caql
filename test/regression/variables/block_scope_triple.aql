// Triple nested block scope test
let x = 10
{
    let y = 20
    {
        let z = 30
        {
            let w = 40
            print("deepest: x=" + x + ", y=" + y + ", z=" + z + ", w=" + w)
        }
        print("middle: x=" + x + ", y=" + y + ", z=" + z)
    }
    print("outer: x=" + x + ", y=" + y)
}
print("global: x=" + x)
