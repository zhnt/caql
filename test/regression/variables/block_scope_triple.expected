deepest: x=10, y=20, z=30, w=40
middle: x=10, y=20, z=30
outer: x=10, y=20
global: x=10

